### ISSUE TITLE
Can you add autocommand event?

### ISSUE THREAD

**Issue by mosheavni** [lukas-reineke/lsp-format.nvim#55]
I want to popup a notification that the file was formatted using client_name, can you either fire a custom autocommand event? or provide an option for a callback using the setup function for after the file is formatted?
Thanks

**Comment by lukas-reineke** [comment]
Would be nice to add something like this :+1:, but I am super busy currently. Probably won't happen for a while, sorry.

**Comment by mosheavni** [comment]
> Would be nice to add something like this 👍, but I am super busy currently. Probably won't happen for a while, sorry.

Sure thing, will you accept a PR?

**Comment by lukas-reineke** [comment]
> will you accept a PR?

Yes, but this is a little more complex than just doing an event when the file is formatted.
There could be more than one client. Each client can have result/no result/error. If there is a result, it might not be applied for different reasons. And each client has a context. Some/All that information should probably be in the event as well somehow.
